include/deal.II/lac/read_write_vector.h

```diff
@@ -206,6 +206,21 @@ namespace LinearAlgebra
     virtual void reinit (const IndexSet &locally_stored_indices,
                          const bool      omit_zeroing_entries = false);
 
+
+#ifdef DEAL_II_WITH_TRILINOS
+#ifdef DEAL_II_WITH_MPI
```

Comment:
What do we do in other places? Shouldn't this function still be visible even without MPI, but just have a disabled implementation?

Reply:
Below we do the same for the ``import`` functions. We never define ``TrilinosWrappers::MPI::Vector`` without Trilinos, so there is no way around that.

Reply:
Ah, great. Thanks for checking.